moxy/org.eclipse.persistence.moxy/src/main/java/org/eclipse/persistence/jaxb/JAXBContextProperties.java

```diff
@@ -29,6 +29,24 @@
  */
 public class JAXBContextProperties {
 
+    /**
+     * The name of the property used to specify which JAXB Context Factory
+     * will be used for context creation.
+     * Default {@code JAXBContextFactory} will be used when this property
+     * is not set.
+     */
+    public static final String MOXY_FACTORY_TYPE = "eclipselink.moxy.factory.type";
```

Comment:
let's call the prop just `eclipselink.moxy.factory` (w/o `type`)

Reply:
done